### PR TITLE
Make read the docs compatible

### PR DESCRIPTION
**Pull request by kdheepak**
Makes docs readthedocs compatible and fix some installation documentation.

### CODE REVIEW

File: docs/installation/mac.md [hunk 1]
```diff
@@ -207,12 +207,12 @@ Run the following in two separate windows.
...
 matlab -nodesktop -nosplash
-cd ~/GitRepos/GMLC-TDC/HELICS-src/swig/matlab
+cd ~/GitRepos/GMLC-TDC/HELICS-examples/matlab
 pireceiver
 ```
 
 ```bash
 matlab -nodesktop -nosplash
-cd ~/GitRepos/GMLC-TDC/HELICS-src/swig/matlab
+cd ~/GitRepos/GMLC-TDC/HELICS-src/matlab
```
Comment (phlptp) [inline docs/installation/mac.md]: should this be examples as well?

File: docs/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+sphinx>=1.4.0,<=1.6.9
+sphinx_rtd_theme
+sphinxcontrib-pandoc-markdown
+ipython
+ipykernel
+nbsphinx
+breathe
```
Comment (phlptp) [inline docs/requirements.txt]: What is this file actually specifying here and who/what is using it?

Comment (kdheepak) [reply docs/requirements.txt]: readthedocs needs this file to know what to install on their end.

File: docs/installation/mac.md (hunk 1 above)
Comment (kdheepak) [reply docs/installation/mac.md]: Yes. Resolved in 0fbf6f3.

**Comment by phlptp** [comment]
Do we need to update the readMe and a few other things to point to a different location?

**Comment by kdheepak** [comment]
Yes we do. We also need to make this similar change on v1.3.x. I think we'll have to make this change on a branch from v1.3.2 and create a new release for v1.3.3. Thoughts?

**Pull request merged**